dev/bin/add-test-users-with-usernames.sh

```diff
@@ -41,8 +44,13 @@ echo "Logging in user $user_c"
 auth_token_c=$(login_user "${user_c}")
 echo "Auth token for $user_c: $auth_token_c"
 
+echo "Logging in user $user_d"
+auth_token_d=$(login_user "${user_d}")
+echo "Auth token for $user_d: $auth_token_d"
```

Comment:
why did you add this? cant you use other users?

Reply:
This user is funded; this is the same user I am using in the fund-user script. I've added the username for that, helpful while developing.